holoviews/core/data/__init__.py

```diff
@@ -64,6 +66,27 @@
 from .. import util
 
 
+def concat(datasets, datatype=None):
+    """
+    Concatenates multiple datasets wrapped in an NdMapping type
+    along all of its dimensions. Before concatenation all datasets
+    are cast to the same datatype. For columnar data concatenation
```

Comment:
'same datatype' determined how?

Reply:
Either explicitly defined or the type of the first dataset that was passed in.

Reply:
Would be good to state that bit about it being chosen from the first one if not explicitly set.